docker_test.go

```diff
@@ -425,7 +553,7 @@ func run(m *testing.M) (int, error) {
 		Tag:          "latest",
 		ExposedPorts: []string{"9090"},
 		Env: []string{
-			"CONFIG_BACKEND_URL=https://api.dev.rudderlabs.com",
+			"CONFIG_BACKEND_URL=https://api.rudderlabs.com",
```

Comment:
Do we really want to do this in production ?

Reply:
Just keeping it for time being, will discuss with Jayc to how to reduce this dependency.